app/src/components/main/attributeList/editableAttributeList.jsx

```diff
@@ -53,8 +59,9 @@ export class EditableAttributeList extends Component {
   };
 
   handleAddNew = () => {
-    const { attributes, onChange } = this.props;
+    const { attributes, onChange, eventsInfo, tracking } = this.props;
     onChange([...attributes, NEW_ATTRIBUTE]);
+    eventsInfo.addAttributeEvent && tracking.trackEvent(eventsInfo.addAttributeEvent);
```

Comment:
This event already tracked in `src/components/main/attributeList/attributeList.jsx`